lib/origen_testers/origen_ext/generator/flow.rb

```diff
@@ -106,6 +108,10 @@ def _create(options = {}, &block)
         job.output_file_body = options.delete(:name).to_s if options[:name]
         if sub_flow
           interface = Origen.interface
+          if reload_target?(interface, options)
+            Origen.app.reload_target!
+            Origen.tester.generating = :program
+          end
```

Comment:
Can you comment on why the reloading has been moved from its original position?

Reply:
@ginty Previously the target was reloaded before the interface was instantiated so there was no chance for the interface to alter the outcome.